doc/SupportedTypes.md

```diff
@@ -14,6 +14,7 @@ Config.Net supports the following CLR types:
 - `System.DateTime`
 - `System.Uri`
 - `System.Guid`
+- `System.Collections.Generic.Dictionary`
```

Comment:
We might need to specify that only specific types of dictionaries are supported. Also that the values should be in json object format?